lib/oxrdf/src/blank_node.rs

```diff
@@ -414,4 +466,64 @@ mod tests {
         assert_eq!(align_of::<BlankNode>(), 8);
         assert_eq!(align_of::<BlankNodeRef<'_>>(), 8);
     }
+
+    #[test]
+    #[cfg(feature = "serde")]
+    fn test_serde() {
+        let b = BlankNode::new_from_unique_id(0x42);
+        let json = serde_json::to_string(&b).unwrap();
+        assert_eq!(json, "{\"value\":\"42\"}");
+        let b2: BlankNode = serde_json::from_str(&json).unwrap();
+
+        assert_eq!(b2, BlankNode::new("42").unwrap());
+
+        match b2 {
+            BlankNode(BlankNodeContent::Anonymous { str: _, id }) => {
+                assert_eq!(id, 0x42u128.to_ne_bytes());
+            }
+            _ => panic!("Expected anonymous blank node"),
+        }
```

Comment:
Do we need that? I think this is already covered by the previous assertion `assert_eq!(b2, BlankNode::new("42").unwrap());`

Reply:
Removed in https://github.com/oxigraph/oxigraph/pull/1272/commits/6b7373010502a2b9d3b423e44ea1b391fa4ccdb7